src/common/log.cpp

```diff
@@ -84,4 +87,42 @@ void log(
     }
 }
 
+auto security_fmt(
+    logging::Severity severity,
+    std::string const& event,
+    std::string const& description) -> std::string
+{
+    auto now = std::chrono::system_clock::now();
+    auto time_t = std::chrono::system_clock::to_time_t(now);
+
+    std::stringstream ss;
+    ss << std::put_time(std::localtime(&time_t), "%FT%TZ");
+
+    return std::format(
+        "{{"
+            "\"datetime\": \"{}\", "
+            "\"appid\": \"mir.canonical.com\", "
```

Comment:
Should this be hardcoded? I don't expect the appid to come from a library the process uses.